cmd/juju/controller/destroy.go

```diff
@@ -123,6 +123,7 @@ var destroySysMsgDetails = `
   - model list: %q;
  - %d machine(s) will be destroyed;
  - %d application(s) will be removed;
+  - application list: %q;
```

Comment:
Also there is a bug here when running against a 3.0.2 controller
```
$ juju destroy-controller lxd-main
WARNING! This command with destroy the "lxd-main" controller and all it's resources:
 - 1 model(s) will be destroyed;
  - model list: "m";
 - 4 machine(s) will be destroyed;
 - 3 application(s) will be removed;
  - application list: "";
 - 0 filesystem(s) and 0 volume(s) will be destroyed.
```

Reply:
Will be fixed with `text/template` introducing